controllers/contacts-controllers.js

```diff
@@ -0,0 +1,84 @@
+const { contactAddSchema } = require("../schemas/contacts-schemas");
+
+const { HttpError } = require("../helpers");
+const ctrlWrapper = require("../decorators/ctrlWrapper");
+const {
+  getContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+  updateStatusContact,
+} = require("../service/DBOperations");
+const { favoriteSchema } = require("../schemas/favorite");
+
+const getAll = async (req, res, next) => {
```

Comment:
```suggestion
const getAllContacts = async (req, res, next) => {
```
аналогічно для інших